### DISCUSSION TITLE
Docker Installation Problems

### DISCUSSION THREAD

**Discussion by joebruin916** [jdholtz/auto-southwest-check-in#267] [General]
When I try to install with Docker, I get this error:

PS C:\Users\Administrator> docker pull jdholtz/auto-southwest-check-in
Using default tag: latest
latest: Pulling from jdholtz/auto-southwest-check-in
no matching manifest for windows/amd64 10.0.17763 in the manifest list entries

Here is my Docker version:

PS C:\Users\Administrator> docker version
Client:
 Version:           26.1.3
 API version:       1.45
 Go version:        go1.21.10
 Git commit:        b72abbb
 Built:             Thu May 16 08:34:37 2024
 OS/Arch:           windows/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          26.1.3
  API version:      1.45 (minimum version 1.24)
  Go version:       go1.21.10
  Git commit:       8e96db1
  Built:            Thu May 16 08:33:14 2024
  OS/Arch:          windows/amd64
  Experimental:     false

Can someone help me with what I am doing wrong?

**Comment by jdholtz** [comment]
The project doesn't currently build any Windows amd64 images (a PR is welcome though, but probably not necessary). The easiest thing to do is switch Docker to use Linux containers (you can find directions [here](https://stackoverflow.com/a/57548944)). Then you should be able to pull the linux/amd64 image. Just note that the script is currently being detected by Southwest in Docker, so you may run into that issue.

**Reply by joebruin916** [reply]
That works on the Desktop version, but not the server version.  I ended up just doing the Python method.

**Reply by jdholtz** [reply]
For the server version, I believe you can run `docker pull jdholtz/auto-southwest-check-in --platform linux/amd64` to get the latest Linux Docker image.
